sardine.go

```diff
@@ -18,7 +20,9 @@ var (
 )
 
 func Run(configPath string) error {
-	conf, err := LoadConfig(configPath)
+	ch := make(chan *cloudwatch.PutMetricDataInput, 1000)
+	mch := make(chan ServiceMetric, 1000)
+	conf, err := LoadConfig(configPath, ch, mch)
```

Comment:
Channels are set into MetricPlugin.PluginDriver.Ch via LoadConfig, but it is difficult to read.

How about pass a channel to PluginDriver after type assertion?

```go
       for _, mp := range conf.MetricPlugins {
               switch d := mp.PluginDriver.(type) {
               case *CloudWatchDriver:
                       d.Ch = ch
               case *MackerelDriver:
                       d.Ch = mch
               }
               go mp.Run(ctx)
        // ...
```